api/execute_test.go

```diff
@@ -17,7 +18,33 @@ import (
 
 func TestAPI_Execute(t *testing.T) {
 
-	srv := setupAPI(t)
+	executionResult := execute.Result{
+		Result: execute.RuntimeOutput{
+			Stdout:   "dummy-failed-execution-result",
+			Stderr:   "dummy-failed-execution-log",
+			ExitCode: 0,
+		},
+	}
+	peerIDs := []peer.ID{
+		peer.ID([]byte{0x0, 0x24, 0x8, 0x1, 0x12, 0x20, 0x56, 0x77, 0x86, 0x82, 0x76, 0xa, 0xc5, 0x9, 0x63, 0xde, 0xe4, 0x31, 0xfc, 0x44, 0x75, 0xdd, 0x5a, 0x27, 0xee, 0x6b, 0x94, 0x13, 0xed, 0xe2, 0xa3, 0x6d, 0x8a, 0x1d, 0x57, 0xb6, 0xb8, 0x91}),
```

Comment:
what does this byte sequence do? can we add comments

Reply:
Oh this is just some sample `peer.ID`, the same one we have in the `mocks` package. Will replace it with that one. So just a value for the variable.